tests/infrastructure/src/main/java/org/mule/test/infrastructure/process/Controller.java

```diff
@@ -68,7 +71,8 @@ public Controller(String muleHome, int timeout) {
     this.muleBin = getMuleBin();
     this.domainsDir = new File(muleHome + "/domains");
     this.appsDir = new File(muleHome + "/apps/");
-    this.libsDir = new File(muleHome + "/lib/user");
+    this.libsDir = new File(muleHome + LIB_USER_FOLDER);
+    this.pluginsDir = new File(muleHome + PLUGINS_FOLDER);
```

Comment:
The compatibility Module should go at App level too??
I follow the logic in the compatibility test done by @elrodro83 https://github.com/mulesoft/mule-ee/blob/mule-4.x/distributions/tests/src/test/java/com/mulesoft/mule/distributions/server/CompatibilityAppControlTestCase.java#L60